docs/de/administration/verwaltung/download-center.md

```diff
@@ -6,4 +6,18 @@ status: new
 lang: de
 ---
 
-# Download Center
+Das Download Center wurde entwickelt, um unseren Kunden den einfachen und direkten Zugang zu den i-doit Updates und Add-on-Paketen für die von ihnen gekauften Produkte zu ermöglichen. Ein Aufrufen einer externen Webseite ist hierfür nicht mehr notwendig.
```

Comment:
```suggestion
Das Download Center wurde entwickelt, um unseren Kunden den einfachen und direkten Zugang zu den i-doit Updates und Add-on-Paketen für die von Ihnen gekauften Produkte zu ermöglichen. Ein Aufrufen einer externen Webseite ist hierfür nicht mehr notwendig.
```

Reply:
ᐠ( ᐛ )ᐟ
In diesem Fall bezieht sich "ihnen" auf "unseren Kunden" und wird als Pronomen in der dritten Person Plural verwendet. Es handelt sich nicht um die formelle Anrede "Ihnen", sondern um die reguläre Verwendung des Personalpronomens "sie" im Dativ.

Reply:
oge